test/tests/parallel/set3/mta/dsn.bats

```diff
@@ -0,0 +1,88 @@
+load "${REPOSITORY_ROOT}/test/helper/setup"
+load "${REPOSITORY_ROOT}/test/helper/common"
+
+BATS_TEST_NAME_PREFIX='[DSN] '
+CONTAINER_NAME='dms-test_dsn'
+
+function teardown() { _default_teardown ; }
+
+@test "should always send a DSN when requested" {
+
+  local LOG_DSN='delivery status notification'
+  local CONTAINER_ARGS_ENV_CUSTOM=(
+    # Required only for delivery via nc (_send_email)
+    --env PERMIT_DOCKER=container
+  )
+
+  _init_with_defaults
+  # Unset `smtpd_discard_ehlo_keywords` to allow DSNs by default on any `smtpd` service:
+  mv "${TEST_TMP_CONFIG}/dsn/postfix-main.cf" "${TEST_TMP_CONFIG}/postfix-main.cf"
```

Comment:
Why us `mv` and not `cp` being used here?

Reply:
Normally we use `mv` as `_init_with_defaults` would copy the config dir into a temporary location for the container being setup (directory with the container name).

Since that is meant to be isolated, the pattern here is just moving from the dsn directory to the root test dir, as this whole directory is mounted into `/tmp/docker-mailserver`.

Not the greatest solution, but that's how it's been for quite some time. I'm not too bothered as it works, and the eventual plan is to switch to `compose.yaml` with `:ro` mounts where possible.

---

`mv` or `cp`, doesn't matter much either way 🤷‍♂️ (_`mv` just communicates the above a bit better_)